README.md

```diff
@@ -27,7 +27,9 @@ Gradle:
 1. [pureconfig](pureconfig) - Module for configuration from [`Config`](https://github.com/lightbend/config).
 1. [extras](extras/README.md) - Module with some extra feature.
 1. [extras-circe](extras-circe/README.md) Adds some circe-dependent functionality.
-1. [extras-cactus](extras-cactus/README.md) Aadds some cactus-dependent functionality.
+1. [extras-cactus](extras-cactus/README.md) Adds some cactus-dependent functionality.
```

Comment:
Why don't you unify all the descriptions? From this, I wouldn't recognize all these three modules are for the "same" thing...

Reply:
This change was just about fixing the type. But I can update the description more, no problem...